### DISCUSSION TITLE
Remove background + mannequin leave apparel. Unreal ?

### DISCUSSION THREAD

**Discussion by TNNSLAB** [ImageMagick/ImageMagick#2162] [Help]
How can I speed up the process of editing this kind of images...any ideas would be appreciated. 

![ASWSLC-BK-1](https://user-images.githubusercontent.com/46636486/84904372-88da6180-b0af-11ea-86a6-2d2df579696a.jpg)

![AW3SS-PU-1](https://user-images.githubusercontent.com/46636486/84904375-88da6180-b0af-11ea-9660-fa2661590da9.jpg)

![ASWFSL-PK-1](https://user-images.githubusercontent.com/46636486/84904341-824bea00-b0af-11ea-988f-1d81aaa1280b.jpg)

even something like this would work : 

![ASWFSL-PK-1](https://user-images.githubusercontent.com/46636486/84904652-d951bf00-b0af-11ea-92b8-b2d9db9239c3.png)

**Comment by fmw42** [comment]
Removing backgrounds is often hard. With some images that have nearly constant color, you can change the background to a uniform color of make it transparent with -fuzz X% -fill white -opaque some_color. But that becomes hard if the main object has similar color.  Use the smallest fuzz value that works.

![redshirt](https://user-images.githubusercontent.com/13458218/84934395-202dbd80-b08c-11ea-9a12-cdd890d950ff.jpg)



`convert redshirt.jpg -fuzz 8% -fill white -opaque white result.jpg`

![result](https://user-images.githubusercontent.com/13458218/84934701-90d4da00-b08c-11ea-8e5f-74d932d8b0e7.jpg)


Usually it is better if your images are not JPG, since that is a lossy compression. But I had to save it as JPG as using PNG made the image too large to upload.
